build.sh

```diff
@@ -17,8 +17,9 @@ usage()
     echo
 }
 
-if [ $1 == "-?" ]; then
+if [ "$1" == "-?" ]; then
     usage
+    exit 0
```

Comment:
hmm now that I think more about it, if we want to have the same behavior as build.cmd we need to keep running the script and pass `-?` to `build-native` and `build-managed`.